src/lib.rs

```diff
@@ -90,6 +90,34 @@ fn get_num_cpus() -> usize {
     }
 }
 
+/// Returns the number of physical CPUs of the current machine.
+#[inline]
+pub fn get_physical() -> usize {
+    get_physical_num_cpus()
```

Comment:
Yea, I'm not sure if it's a good idea to expose this, and panic on unsupported platforms, or to only expose this method on supported platforms. In other words, be a runtime panic, or be a compile time error.